packages/react-sdk/src/components/index.test.ts

```diff
@@ -1,9 +1,12 @@
 import * as components from "./index";
 import { WsComponentMetaSchema } from "./component-type";
 
-test.each(Object.entries(components))("validating meta definition of %s", (name, component) => {
-  if (name === "default") {
-    return;
+test.each(Object.entries(components))(
```

Comment:
What happened here, either your editor or @rpominov's isn't formatting using the same prettier settings

Reply:
I think we don't have a prettier config at all :)
So everyone runs with their own settings probably.

Reply:
Ah, this might actually be because I've accepted a suggested change in the PR comment: https://github.com/webstudio-is/webstudio-designer/commit/31e710efca594527b5d644da70b95ef4dcc11dae

Reply:
that makes sense